src/components/ViewRequest/Actions/SendRequest.tsx

```diff
@@ -72,7 +80,20 @@ const SendRequest = () => {
         newData.newSubStage = nextStage?.subStages?.[0].key || ""; // first substage of next stage or empty string
         newData.eventTitle = currentStage.nextEventTitle;
       }
-      updateStage.mutate(newData);
+      updateStage.mutateAsync(newData).then(() => {
+        if (currentStage?.key === "Recruiting") {
+          addNote.mutate("Moved to Candidate Selection");
+        }
+        if (currentStage?.key === "Selection") {
+          let myString = "Moved to Package Prep and Approval.\n";
```

Comment:
This isn't being rendered in the note as a newline.  Perhaps you need a carraige return (\r) too?